generate_params.py

```diff
@@ -0,0 +1,915 @@
+                            use_transformer=True,
+                            use_bert=True,
+                            gcs_bucket='sequin-public',
+                            random_keys=range(10))
+
+    # Pretrained medium transformer
+    params += create_params(
+        encoder_lrs=[0.0],
+        lens_lrs=[1e-5],
+        predictor_lrs=[1e-3],
+        encoder_wds=[0.0],
+        lens_wds=[0.05],
+        predictor_wds=[0.0],
+        reduce_fn_kwargs_paths=['linear_pool_1024'],
+        lens_train_samples=[50],
+        train_families=10000,
+        epochs=10,
+        measurements=2,
+        encoder_fn_name='transformer',
+        encoder_fn_kwargs_path='medium_transformer_kwargs',
+        reduce_fn_name='linear_max_pool',
+        lens_batch_size=64,
+        knn_batch_size=64,
+        use_transformer=True,
+        use_bert=True,
+        restore_transformer_dir=
+        'gs://sequin-public/transformer_models/medium_trembl_bert/',
+        gcs_bucket='sequin-public',
+        random_keys=range(10))
+
+    # Medium transformer
+    params += create_params(encoder_lrs=[0.0],
+                            lens_lrs=[5e-5],
+                            predictor_lrs=[5e-4],
+                            encoder_wds=[0.0],
+                            lens_wds=[0.2],
+                            predictor_wds=[0.2],
+                            reduce_fn_kwargs_paths=['linear_pool_1024'],
+                            lens_train_samples=[50],
+                            train_families=10000,
+                            epochs=10,
+                            measurements=2,
+                            encoder_fn_name='transformer',
+                            encoder_fn_kwargs_path='medium_transformer_kwargs',
+                            reduce_fn_name='linear_max_pool',
+                            lens_batch_size=64,
+                            knn_batch_size=64,
+                            use_transformer=True,
+                            use_bert=True,
+                            gcs_bucket='sequin-public',
+                            random_keys=range(10))
+
+    # Pretrained medium transformer
+    params += create_params(
+        encoder_lrs=[0.0],
+        lens_lrs=[5e-5],
+        predictor_lrs=[5e-4],
+        encoder_wds=[0.0],
+        lens_wds=[0.2],
+        predictor_wds=[0.2],
+        reduce_fn_kwargs_paths=['linear_pool_1024'],
+        lens_train_samples=[50],
+        train_families=10000,
+        epochs=10,
+        measurements=2,
+        encoder_fn_name='transformer',
+        encoder_fn_kwargs_path='medium_transformer_kwargs',
+        reduce_fn_name='linear_max_pool',
+        lens_batch_size=64,
+        knn_batch_size=64,
+        use_transformer=True,
+        use_bert=True,
+        restore_transformer_dir=
+        'gs://sequin-public/transformer_models/medium_trembl_bert/',
+        gcs_bucket='sequin-public',
+        random_keys=range(10))
+
+    # Small transformer
```

Comment:
there are many blocks with the same name, such as 'Small transformer.' What is the difference between them?

Reply:
made some comments that group them by regular sweep, random sweep, and transformer size